setup.py

```diff
@@ -46,6 +51,11 @@
         # $ pip install jax[cpu]
         'cpu': [f'jaxlib>={_minimum_jaxlib_version}'],
 
+        # Cloud TPU VM jaxlib can be installed via:
+        # $ pip install jax[tpu]
+        'tpu': [f'jaxlib=={_current_jaxlib_version}',
+                f'libtpu-nightly @ {_libtpu_url}'],
```

Comment:
I haven't seen this `<libary> @ <url>` syntax before. Could we use the same thing to avoid requiring the -f addition for GPU installs?

Reply:
@yashk2810 pointed me to it! https://stackoverflow.com/questions/12518499/pip-ignores-dependency-links-in-setup-py/54216163#54216163

And yes, I think we could use it for the GPU builds, great idea! It would require hardcoding the wheel URL, but that doesn't seem bad given we already hardcode the jaxlib version. Let's do it in a follow-up change though, as I'd like to get this in ASAP to avert impending jaxlib/libtpu incompatibility.